docs/source/flow-project.rst

```diff
@@ -92,6 +90,11 @@ A very simple *operation*, which creates a file called ``hello.txt`` within a jo
     if __name__ == "__main__":
         MyProject().main()
 
+.. tip::
+
+    By default operations only act on a single job and can simply be defined with the signature `def op(job)`.
```

Comment:
Use code font rather than italics.
```suggestion
    By default operations only act on a single job and can simply be defined with the signature ``def op(job)``.
```